localstack/aws/api/core.py

```diff
@@ -221,7 +221,9 @@ def service_operation(self) -> ServiceOperation:
 ServiceRequestHandler = Callable[[RequestContext, ServiceRequest], Optional[ServiceResponse]]
 
 
-def handler(operation: str = None, context: bool = True, expand: bool = True):
+def handler(
```

Comment:
The handler will have an extra property in the marker to signal the implementation is in the provider, for the cases we want to add functionality, for example, custom implementations not in moto.